src/payment/payment-methods/stripe/service.js

```diff
@@ -56,17 +71,20 @@ export default async function checkout(
           purchased_for_organization: purchasedForOrganization,
         },
       },
+      // Shipping is required for processing Afterpay payments
+      shipping: shippingAddress,
     },
   });
 
-  if (result.error?.code === 'payment_intent_unexpected_state' && result.error?.type === 'invalid_request_error') {
+  if (result.error) {
```

Comment:
previously this was checking for the presence of `.error` , just want to double check that that's no longer required

Reply:
Still checking for the presence of `result.error` here, I modified this because this calls `handleApiError()` which calls `handleRequestError()' and the type of error is handled there. 

So the original code was just being more specific to one type of error and now I want to be more broad, but at the end the same type of error will still be handled.